packages/components/src/components/form/Select/index.tsx

```diff
@@ -25,8 +26,11 @@ const selectStyle = (
         display: 'flex',
         alignItems: 'center',
         width: '100%',
+        margin: 0,
+        padding: '0 8px',
```

Comment:
may that brake the layout in other implementations of Select all over the suite?
i let it @slowbackspace to decide :)

Reply:
@tomasklim please consult Ondra if these are the changes we want to make globally for all selects and verify it didn't broke anything. For example selects are used in send form - for selecting a token or fiat currency, also on few places in settings, add-account modal, I can check the select for recovery on onboarding for you :). Especially change to font-size and font-weigh is looking suspicious, but it indeed might be what we want if Ondra confirms it 😀

Reply:
@slowbackspace ondrej agreed to remove the green check mark for now and confirms that these changes should be across the whole app